### ISSUE TITLE
MC 1.18.2

### ISSUE THREAD

**Issue by winnetrie83** [eerussianguy/Better-Foliage#14]
As of version: betterfoliage-3.0.3.jar,

Biomesoplenty and quark leaves are not working. They appear normal.
Only vanilla leaves are working fine

**Comment by eerussianguy** [comment]
Do not spam my github

**Comment by winnetrie83** [comment]
> Do not spam my github

I did not such a thing. 
I opened a new issue (this one) and I mentioned it in an already closed post.
I'm sorry if you consider this "spam"

Since you closed that older post and this new 1 without any reason, I guess you won't fix the issue. It's ok if you won't, but just say it.

In stead of being rude to people reporting an issue, you could have given a serious answer.

**Comment by eerussianguy** [comment]
You are clearly smart enough to see that I closed a duplicate issue on this in the past and yet you started leaving comments everywhere and opened a new issue. I get messages about BOP literally every single day. I am working on it.

**Comment by eerussianguy** [comment]
I have identified the issue and it will be fixed in the next release

**Comment by winnetrie83** [comment]
Then instead of closing it, let it open and tell people you are working on it.

If you provide 10 platforms where people can place a comment, then i'll probably use 5 of them to place a comment. That's not spam, that's trying to be helpfull.

FYI I placed a comment on curseforge, wich you have blocked apparently. I opened a new issue on your git, because this issue was still there and no other was still open marked as "working on it" or something. 

I did also mentioned it in an older already closed issue, but decided to betteropen a new 1.

I'm glad you see i'm smart.
So by this i'll give you a warning. 
I'll report you for trolling. And don't worry, I have everything screenshotted as proof. 
You made a big mistake to harras the "smart" guy.

People like you makes me sick.
